bin/opt8n/src/opt8n.rs

```diff
@@ -124,11 +134,12 @@ impl Opt8n {
     }
 
     async fn receive_command(&self) -> Result<ReplCommand> {
-        let line = BufReader::new(tokio::io::stdin())
-            .lines()
-            .next_line()
-            .await?
-            .unwrap();
+        let line = tokio::task::spawn_blocking(|| {
+            inquire::Text::new("Enter a command")
+                .with_help_message("'help' for more information")
+                .prompt()
+        })
+        .await??;
```

Comment:
I think we can probably display "Enter a command" once at the beginning. Something like the following

```bash
```bash
Welcome to Opt8n! Type `!help` to show available commands.
➜ 
# some command 
➜ 
```

What do you think about modeling the repl UX after `chisel`? 
[Here is a link to chisel in case you havent used it before!](https://github.com/foundry-rs/foundry/tree/master/crates/chisel).

Reply:
This would be very nice, chisel should be a good reference to go off of.